WebRoot/messages/ZaMsg.properties

```diff
@@ -2332,6 +2332,9 @@ ss_inactive_accounts_30 = Inactive Accounts (30 days)
 ss_inactive_accounts_90 = Inactive Accounts (90 days)
 
 #Busy Messages
+BUSY_LOADING = Loading...
+BUSY_TESTING = Testing...
+BUSY_CREATING = Creating...
```

Comment:
BUSY_TESTING and BUSY_CREATING are not used.

Reply:
They are used in this PR: https://github.com/Zimbra/zm-hsm-admin-zimlet/pull/29

Reply:
It should be passed as `label` from a caller.
```
loaderTitle.innerHTML = label ? label : ZaMsg.BUSY_LOADING;
```
We can remove the two from here.

Reply:
I reconsidered. That's fine. Please ignore my previous comment.